2_base_and_sensor.ipynb

```diff
@@ -214,28 +396,47 @@
     "</div>"
    ]
   },
+  {
+   "cell_type": "markdown",
+   "metadata": {
+    "lang": "en"
+   },
+   "source": [
+    "<div class=\"alert alert-block alert-info\">\n",
+    "    <b>Task</b>\n",
+    "    <p>Let's write a code to move the robot from the initial position to the front of the human.</p>\n",
+    "    <p>By entering the following command, we can move the robot to the initial position.</p>\n",
```

Comment:
If it is one command, then it is OK not to change the sentence.
But, if there are multiple commands, then the following is more correct:

    "    <p>By entering the following commands, we can move the robot to the initial position.</p>\n",